src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,40 @@
+.nav {
+  &__list {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    list-style: none;
+    font-size: 12px;
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    height: 60px;
+    align-items: center;
+    text-decoration: none;
+    color: $color-main-accent;
+    font-weight: 500;
+
+    &:hover {
+      color: $color-blue-accent;
+    }
+
+    &--active {
+      color: $color-blue-accent;
+
+      &::after {
+        position: absolute;
```

Comment:
this is absolute to which element or class?
i think it should be to `nav__link`

Reply:
It's resposible for a blue bar under Apple nav-link--active. My understanding was it should only be under Apple link. No?

Reply:
Got it.